syft/pkg/cataloger/dotnet/cataloger.go

```diff
@@ -4,10 +4,13 @@ import (
 	"github.com/anchore/syft/syft/pkg/cataloger/generic"
 )
 
-const catalogerName = "dotnet-deps-cataloger"
-
 // NewDotnetDepsCataloger returns a new Dotnet cataloger object base on deps json files.
 func NewDotnetDepsCataloger() *generic.Cataloger {
-	return generic.NewCataloger(catalogerName).
+	return generic.NewCataloger("dotnet-deps-cataloger").
 		WithParserByGlobs(parseDotnetDeps, "**/*.deps.json")
 }
+
+func NewDotnetPortableExecutableCalaloger() *generic.Cataloger {
```

Comment:
```suggestion
func NewDotnetPortableExecutableCataloger() *generic.Cataloger {
```

Reply:
😆

Reply:
--> https://github.com/anchore/syft/pull/1943/commits/e42fd77b2f8b6d42e076a84f6cce386861260941

Reply:
But Syft should **absolutely** revisit this `Calaloger` concept when it shifts its focus to Song Composition Analysis 🎵